drivers/SmartThings/philips-hue/profiles/plug.yml

```diff
@@ -0,0 +1,12 @@
+name: plug
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: samsungim.hueSyncMode
```

Comment:
Can the plug be used with hue sync?

Reply:
It's unclear to me what exactly it would do, but the representation does return a `mode` value when queried.

Reply:
OK, probably does make sense to leave it in then